From 7a2ff251677fe00690b5bd27b85c418a81099f75 Mon Sep 17 00:00:00 2001
From: Han Wang <416810799@qq.com>
Date: Sat, 8 Jun 2019 07:42:50 +0200
Subject: [PATCH 5/5] lineage-sdk: Disable color temperature adjustment by
 default

 * NightLight is disabled by default and it is better to keep
   sync between HWC2 and non-HWC2 devices.

Change-Id: Ibac762384c38dd33c9e6fdeb41e1774314fffe1a
---
 lineage/res/res/values/config.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lineage/res/res/values/config.xml b/lineage/res/res/values/config.xml
index b86a129..8a40fa7 100644
--- a/lineage/res/res/values/config.xml
+++ b/lineage/res/res/values/config.xml
@@ -30,7 +30,7 @@
     <integer name="config_nightColorTemperature">4800</integer>
     <integer name="config_outdoorAmbientLux">12000</integer>
     <integer name="config_outdoorAmbientLuxHysteresis">1500</integer>
-    <integer name="config_defaultLiveDisplayMode">2</integer>
+    <integer name="config_defaultLiveDisplayMode">0</integer>
 
     <!-- These values should map to the true min and max
          that the backend is capable of adjusting to. This
-- 
2.17.1

